app/utils/load-helpscout-beacon.js

```
@@ -0,0 +1,74 @@
+export function loadScript(src) {
```

ESLint

[error] 'src' is defined but never used.

(no-unused-vars)